sapi/fpm/tests/bug80849-fpm.phpt

```diff
@@ -0,0 +1,10 @@
+--TEST--
```

Comment:
This is just testing the CGI binary again.

sapi/fpm/tests/bug69625-no-script-filename.phpt is probably how the FPM test should look like.

Reply:
Ah, thanks, I'll have a look.

Reply:
I fixed the test.